frontend/src/metabase/nav/containers/Navbar.jsx

```diff
@@ -361,10 +398,29 @@ export default class Navbar extends Component {
           )}
           <ProfileLink {...this.props} />
         </Flex>
+        {this.renderModal()}
```

Comment:
Outside the scope of this PR, but maybe this should be a HOC? You'd wrap a component and it would receive a `setModal` prop and have the modal rendered after it. We'd probably want a central modal registry with that.

Reply:
Yeah honestly this should probably just be refactored to use the `ModalRoute` instead of this. For the purposes of these changes I just wanted to set things back to their known state, but I do think some cleanup is in order to your point.